pixie-uefi/src/flash.rs

```diff
@@ -103,37 +102,14 @@ pub async fn flash(os: UefiOS, server_addr: SocketAddrV4) -> Result<()> {
     }));
 
     let stats2 = stats.clone();
-    os.set_ui_drawer(move |os| {
-        os.write_with_color(
-            &format!("{} total chunks\n", stats2.borrow().chunks),
-            Color::White,
-            Color::Black,
-        );
-        os.write_with_color(
-            &format!("{} unique chunks\n", stats2.borrow().unique),
-            Color::White,
-            Color::Black,
-        );
-        os.write_with_color(
-            &format!("{} chunks to fetch\n", stats2.borrow().fetch),
-            Color::White,
-            Color::Black,
-        );
-        os.write_with_color(
-            &format!("{} chunks received\n", stats2.borrow().recv),
-            Color::White,
-            Color::Black,
-        );
-        os.write_with_color(
-            &format!("{} packets received\n", stats2.borrow().pack_recv),
-            Color::White,
-            Color::Black,
-        );
-        os.write_with_color(
-            &format!("{} chunks requested\n", stats2.borrow().requested),
-            Color::White,
-            Color::Black,
-        );
+    crate::os::ui::set_content_drawer(move |draw_area| {
+        draw_area.clear();
+        draw_area.writeln(&format!("{} total chunks", stats2.borrow().chunks));
```

Comment:
We could implement `core::fmt::Write` so we can do
```suggestion
        writeln!(draw_area, "{} total chunks", stats2.borrow().chunks);
```